packages/pwa/app/components/_app/index.jsx

```diff
@@ -273,7 +274,7 @@ App.shouldGetProps = () => {
 }
 
 App.getProps = async ({api, res}) => {
-    const l10nConfig = getL10nConfig(res.locals.originalUrl)
+    const l10nConfig = resolveSiteFromUrl(pathToUrl(res.locals.originalUrl))?.l10n
```

Comment:
There is no shame in adding lines of code 🤪 . The goal of our code should be simplicity and readability, not minimal lines of code. This way it's readable and understandable by the partners that use this template. 

I'd break things about a little: 

```
const site = pathToUrl(res.locals.originalUrl)
const l10nConfig = site.l10n
```